Assets/Mirror/Components/Discovery/NetworkDiscoveryBase.cs

```diff
@@ -30,7 +30,11 @@ public abstract class NetworkDiscoveryBase<Request, Response> : MonoBehaviour
         [SerializeField]
         [Tooltip("The UDP port the server will listen for multi-cast messages")]
         protected int serverBroadcastListenPort = 47777;
-
+        
+        [SerializeField]
+        [Tooltip("")]
+        bool enableActiveDiscovery = true;
```

Comment:
if you make this public instead of serializefield then the user's game code can enable/disable it too